Biblio.bib

```diff
@@ -75,14 +75,15 @@ @Misc{EIP-2
 }
 
 @Misc{cryptoeprint:2013:881,
-	Note = "\url{http://eprint.iacr.org/}",
-	Url = "Cryptology ePrint Archive, Report 2013/881",
+	url = "https://eprint.iacr.org/2013/881",
 	author = "Sompolinsky, Yonatan and Zohar, Aviv",
 	title = "Accelerating Bitcoin{'}s Transaction Processing. Fast Money Grows on Trees, Not Chains",
 	year = "2013",
 }
 
 @InCollection{gura2004comparing,
+	url = "https://www.iacr.org/archive/ches2004/31560117/31560117.pdf",
+	note = "\url{https://web.archive.org/web/20170810035057/https://www.iacr.org/archive/ches2004/31560117/31560117.pdf}",
```

Comment:
I don't think the web archive link is necessary.  When the above link is not working anymore, people can look into web archive.

Reply:
OK, fair enough. I guess that I was thinking it would be better to be proactive and include the Web archive link, so that if the original died, another change would not have to be made.